app/views/users/_buttons.html.haml

```diff
@@ -5,9 +5,8 @@
   .btn.btn-default{:title=>"#{user.name} works on #{user.projects.length} projects"}
     = user.projects.length
     %i.fa.fa-puzzle-piece
-  / FIXME. Use real count
-  .btn.btn-default{:title=>"#{user.name} wrote #{user.projects.length} comments"}
-    0
+  .btn.btn-default{:title=>"#{user.name} wrote #{Comments.where(commenter_id: user).count} comments"}
```

Comment:
This should be `Comment` instead of `Comments`. 
BTW, this could be also written as: `user.comments.size` and in the relationship with comments we should add the foreign_key. Something like `has_many :comments, foreign_key: 'commenter_id'`